ClientApp/src/app/services/indicator-group/indicator-group.service.ts

```diff
@@ -24,6 +25,10 @@ export class IndicatorGroupService {
     return this.http.get<IndicatorGroup[]>(IndicatorGroupService.API_URL);
   }
 
+  getIndicatorGroupsComplete(): Observable<IndicatorGroup[]> {
```

Comment:
This definition conflicts with a previous definition of the function when compared to the next pull request. Please check this, but this is no problem to merge at this point.